mkdocs.yml

```diff
@@ -6,7 +6,7 @@ theme:
   palette:
     scheme: preference
   features:
-    - instant
+    - navigation.instant
```

Comment:
This fixes the navigation not being instant - fairly sure the docs might've changed or something.

Reply:
https://squidfunk.github.io/mkdocs-material/upgrading/#upgrading-from-5x-to-6x